src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -78,6 +78,9 @@ export default class DataTable extends Vue {
   showExpandAll = Object.prototype.hasOwnProperty.call(this.$props.config, 'showExpandAll')
     ? this.$props.config.showExpandAll
     : defaultConfig.showExpandAll;
+  showSelectAllDropdown = Object.prototype.hasOwnProperty.call(this.$props.config, 'showSelectAllDropdown')
+    ? this.$props.config.showSelectAllDropdown
+    : defaultConfig.showSelectAllDropdown;
```

Comment:
Can you test this? Use the `showSelectAllDropdown` with value `true` and `false` and also without assigning it in the configuration and check that it has the same behaviour.

```suggestion
  showSelectAllDropdown = !!this.$props.config.showSelectAllDropdown;
```

Reply:
You're right, but other config properties are defined this way:
```
  treeSelection = Object.prototype.hasOwnProperty.call(this.$props.config, 'treeSelection')
    ? this.$props.config.treeSelection
    : defaultConfig.treeSelection;
  cellWrap = Object.prototype.hasOwnProperty.call(this.$props.config, 'cellWrap')
    ? this.$props.config.cellWrap
    : defaultConfig.cellWrap;
  showExpandAll = Object.prototype.hasOwnProperty.call(this.$props.config, 'showExpandAll')
    ? this.$props.config.showExpandAll
    : defaultConfig.showExpandAll;
```

So I've defined this one same as the rest. Do you still want me to change this variable?